docs/java-agent-injector.md

```diff
@@ -44,14 +44,20 @@ inject-demo   1/1     Running   0          2d2h
 
 The java agent injector supports a precedence order to configure the agent:
 
-``` Annotations > Configmap > Default configmap```
+``` Annotations > SwAgent CR > Configmap > Default configmap```
```

Comment:
I highly recommend to remove the namespaced configmap which can be substituted by `SwAgent`. If we though this pruning will affect too many pieces, a `deprecated` label should be attached to it.

Reply:
what do you think about:


``` Annotations > SwAgent > default config ```

of 

``` Annotations > SwAgent CR > Configmap (deprecated) > default config```

Reply:
I think removing agent.config configmap would cause many changes. i don't know if simply removing the `skywalking-swck-java-agent-configmap` volume would work. it looks like that we should also add the default file into agent image.  Maybe we can open another issue to discuss about it.